src/java/fr/paris/lutece/plugins/forms/modules/documentproducer/utils/PDFUtils.java

```diff
@@ -383,8 +386,8 @@ private static void builderPDFWithFormQuestionResponse( Document document, FormR
                         if ( formQuestionResponseOfStep.getEntryResponse( ).size( ) == 1 )
                         {
                             // One element, build a paragraph
-                            String strValue= EntryTypeServiceManager.getEntryTypeService( questionOfStep.getEntry( ) ).getResponseValueForRecap( questionOfStep.getEntry( ), null, formQuestionResponseOfStep.getEntryResponse( ).get( 0 ), Locale.FRENCH );
-                            Paragraph paragraphResponse = new Paragraph( new Phrase( strValue, fontResponse ) );
+                            Paragraph paragraphResponse = new Paragraph( new Phrase( formQuestionResponseOfStep.getEntryResponse( ).get( 0 )
+                                    .getToStringValueResponse( ), fontResponse ) );
```

Comment:
How does this change the output ? this does not seem specific to address fields...